lib/apple/index.js

```diff
@@ -35,7 +35,12 @@ function parseResult(result) {
 	var receipt = result.receipt;
 
 	return {
+		status: result.status,
```

Comment:
I have to agree that that's a bit against the original philosophy (where properties at the top level are universal, regardless of which store is being used). We can either change the philosophy to say "at least properties A, B, C, optionally augmented with store specific props", or we put all of these in another sub-object.

@HolgerFrank any input on this?